pom.xml

```diff
@@ -2333,6 +2333,12 @@
             <artifactId>log4j-web</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+        <!-- This is needed to support JSON logging output for DataDog -->
```

Comment:
I agree we need to have a comment here but IMHO this looks to much like a KualiCo comment and not a Cornell customization. I think it needs to be more like the comment style on lines 29-31 of this file. Maybe something like the following which would notate why we took the log4j version bump so that is not lost during future KualiCo upgrades (since we will be ahead of the version they are using).

_CU customization to support JSON logging for DataDog. Log4j minimum version 2.21.1 required for this feature._

Thoughts?

Reply:
no objections to comment change, I shall change